pom.xml

```diff
@@ -37,9 +37,9 @@
     <inceptionYear>2017</inceptionYear>
 
     <properties>
-        <version.thorntail>2.2.1.Final</version.thorntail>
-        <maven.compiler.source>1.8</maven.compiler.source>
-        <maven.compiler.target>1.8</maven.compiler.target>
+        <version.thorntail>2.5.0.Final</version.thorntail>
```

Comment:
CQ For Thorntail 2.5.0.Final is rejected, we are not allowed to upgrade to this version. 
https://dev.eclipse.org/ipzilla/show_bug.cgi?id=20870

(even 2.2.1 was not allowed https://dev.eclipse.org/ipzilla/show_bug.cgi?id=18884)

Reply:
@rdebusscher Thorntail has never passed the Eclipse CQ as such IIRC.   @cealsair  and @Ladicek  promised to follow up on it...

Reply:
2.6.0.Final should have all the license headers sorted out.